api/src/main/java/io/opencensus/trace/Ctx.java

```diff
@@ -0,0 +1,6 @@
+package io.opencensus.trace;
+
+public interface Ctx {
```

Comment:
I know I came up with this naming (sorry), but it might better to call this interface `Context` or `ContextHandle`?
Then we can also rename `CtxUtils -> ContextHandleUtils` or similar.

Keeping `Ctx` is also fine - whatever you feel like is better :)